contracts/instruments/RibbonThetaVault.sol

```diff
@@ -112,13 +123,23 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
 
         asset = _isPut ? _usdc : _asset;
         underlying = _asset;
+        collateralToken = IYearnVault(
+            _isYearnWrapped
+                ? IYearnRegistry(_yearnRegistry).latestVault(
```

Comment:
in the case that this isnt a yearn wrapped vault, I still instantiate the yearn vault contract but pass in asset instead of yAsset. we don't use deposit/withdraw so it could be misleading but it all still works